io/module_write_restart_netcdf.F90

```diff
@@ -346,13 +347,17 @@ subroutine write_restart_netcdf(wrtfb, filename, &
          else if (rank == 3) then
            if ( .not.is_restart_core ) then
              dimids_3d = [im_dimid,jm_dimid,zaxis_dimids(i),time_dimid]
+             chunksizes = [im, jm, 1, 1]
```

Comment:
Does this mean the chunksize is fixed to be [im,jm,1,1]? Do we allow user defined chunksize through configuration file? If yes, this can be done later though.

Reply:
Yes. The chunksize for the restart files is (currently) fixed to the size of horizontal dimensions.